api/tests/unit-tests/backend/core/test_geometry.py

```diff
@@ -56,13 +56,10 @@ def test__convert_raster_to_box():
     subquery1 = (
         select(
             models.Annotation.id.label("id"),
-            func.ST_MakeValid(
-                type_coerce(
-                    func.ST_DumpAsPolygons(models.Annotation.raster),
-                    GeometricValueType(),
-                ).geom,
+            func.ST_Envelope(
```

Comment:
This is a patch so it could technically go in another pr

Reply:
I am working on broader testing in this issue https://github.com/Striveworks/valor/issues/466

Reply:
in that case, you should probably move these changes to another PR and link that PR to #466

Reply:
This is blocking for this pr

Reply:
> This is a patch so it could technically go in another pr

> This is blocking for this pr

I'm confused; why is this change necessary for a PR about embeddings?

Reply:
addressed in https://github.com/Striveworks/valor/pull/469